### ISSUE TITLE
UpdateFrequency / RenderFrequency not working (v4)

### ISSUE THREAD

**Issue by arakis** [opentk/opentk#1029]
### Description

`GameWindow.UpdateFrequency` and `GameWindow.RenderFrequency` are not respected, they are always at full speed. Only VSYNC can throttle the  `RenderFrequency` to 60. This is regardless of the `IsMultiThreaded` flag.

### Repro steps

Set `GameWindow.UpdateFrequency` to 10.0.
Set `GameWindow.RenderFrequency` to 10.0.

Measurement Helper:

```c#
public class EventCounter
{
    private Stopwatch Watch = new Stopwatch();

    private double FrameCount = 0;
    private double DeltaTime = 0.0;
    private double UpdateRate = 4.0;  // 4 updates per sec.        

    private double _Fps = 0.0;
    public double EventsPerSecond => _Fps;

    public void Tick()
    {
        Watch.Stop();
        Elapsed = Watch.Elapsed;
        FrameCount++;
        DeltaTime += Watch.ElapsedMilliseconds / 1000.0;
        if (DeltaTime > 1.0 / UpdateRate)
        {
            _Fps = FrameCount / DeltaTime;
            FrameCount = 0;
            DeltaTime -= 1.0 / UpdateRate;
        }
        Watch.Restart();
    }
}
```

### Expected behavior

Round about (in delta) 10 Events per Second.

### Actual behavior

It's not throttled, regardless of IsMultiThreaded. Only VSYNC=On/Auto can throttle RenderFrequency to 60 (but not below). 

### Related information 

* Linux Gentoo, 64 Bit.
* OpenTK builded from latest master
* dotnet core 3.1

**Comment by MerickOWA** [comment]
This should be fixed now in 4.0.0-pre6

**Comment by arakis** [comment]
I close this issue, because it's fixed in #1038. Thx to @MerickOWA
